### ISSUE TITLE
Fix Resource Warning

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8647]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8647|
|Type|enhancement|
|Created|2016-07-21 13:33:57Z|

Runing tests with tox reveals this warning

```
/home/ac/Twisted2/build/py35-tests/lib/python3.5/site-packages/twisted/internet/test/connectionmixins.py:406: ResourceWarning: unclosed file 
<_io.TextIOWrapper name='twisted.internet.test.test_addre/UNIXAddressTests/test_hashOfLinkedFiles/uo4gdke0/temp' mode='w' encoding='UTF-8'>
```

<details><summary>Searchable metadata</summary>

```
trac-id__8647 8647
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469108037581314 1469108037581314
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

See https://github.com/twisted/twisted/pull/382

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset e74d32c3867e71936b1c3464280ced85994d06e0
```
#!CommitTicketReference repository="" revision="e74d32c3867e71936b1c3464280ced85994d06e0"
Merge pull request #388 from twisted/8647-acabhishek942-resourcewarning

Author: acabhishek942
Reviewer: rodrigc
Fixes: #8647

Close more files
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
